extension/test/e2e/pageObjects/AffirmPage.js

```diff
@@ -0,0 +1,31 @@
+module.exports = class AffirmPage {
+  constructor(page) {
+    this.page = page
+  }
+
+  async finishAffirmPayment() {
+    // Input phone number and proceed
```

Comment:
Tip: instead of commenting, you can extract the calls into methods and name them like your comments. It's always better than comments.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/769/commits/1552c81499c5037091d812d0d3235c35df27efe6